src/information_theory/hamming.lean

```diff
@@ -84,14 +87,14 @@ by rw [eq_comm, hamming_dist_eq_zero]
 
 /-- Corresponds to `dist_ne_zero`. -/
 lemma hamming_dist_ne_zero {x y : Π i, β i} : hamming_dist x y ≠ 0 ↔ x ≠ y :=
-not_iff_not.mpr hamming_dist_eq_zero
+not_congr hamming_dist_eq_zero
```

Comment:
```suggestion
hamming_dist_eq_zero.not
```
if you wanted to get even shorter

Reply:
https://github.com/leanprover-community/mathlib/pull/15820#discussion_r938640486